cypress/support/functions.ts

```diff
@@ -168,3 +168,53 @@ Cypress.Commands.add('deleteNamespace', ({namespace, appName}) => {
     cy.contains(appName).should('not.exist');
   }
 });
+
+// Epinio installation functions
+
+// Add the Epinio Helm repo
+Cypress.Commands.add('addHelmRepo', ({repoName, repoUrl}) => {
+  cy.get('.nav').contains('Apps & Marketplace').click();
```

Comment:
As these 2 commands are used multiple time it might be a good idea to either create a function (or modify an existing one if it's possible, not sure).

Reply:
Yes, I can try to add a function in the uninstallation test because I will need it there as well.